SDL/Event.cu

```diff
@@ -806,13 +806,13 @@ void SDL::Event::createSegmentsWithModuleMap()
     for (int i=0; i<nLowerModules; i++) {
       unsigned int innerLowerModuleIndex = modulesInGPU->lowerModuleIndices[i];
       unsigned int nConnectedModules = modulesInGPU->nConnectedModules[innerLowerModuleIndex];
-      unsigned int nInnerMDs = nMDs[innerLowerModuleIndex];
+      unsigned int nInnerMDs = nMDs[innerLowerModuleIndex] > N_MAX_MD_PER_MODULES ? N_MAX_MD_PER_MODULES : nMDs[innerLowerModuleIndex];
```

Comment:
```suggestion
      unsigned int nInnerMDs = std::min(nMDs[innerLowerModuleIndex],N_MAX_MD_PER_MODULES);
```
is shorter.
However, is there a `-1` missing?

Reply:
>  However, is there a `-1` missing?

no, probably not

Reply:
BTW, if the checks are done at write time, perhaps they are not really required at read time